ChecksFinderClient.py

```diff
@@ -32,6 +32,14 @@ def __init__(self, server_address, password):
         self.send_index: int = 0
         self.syncing = False
         self.awaiting_bridge = False
+        # self.game_communication_path: files go in this path to pass data between us and the actual game
+        if "localappdata" in os.environ:
+            self.game_communication_path = os.path.expandvars(r"%localappdata%/ChecksFinder")
+        else:
+            # not windows. game is an exe so maybe the user wants to use wine?
+            self.game_communication_path = os.path.join(
+                os.path.expanduser("~/.wine/drive_c"),
```

Comment:
this should allow to use a custom WINEPREFIX

Reply:
Added, thanks!